bindings/rust/bench/src/harness/mod.rs

```diff
@@ -325,93 +311,6 @@ where
 
         Ok(())
     }
-
-    /// Shrink buffers owned by the connections
-    pub fn shrink_connection_buffers(&mut self) {
```

Comment:
Why did we have these shrink functions? How are we able to just delete them without having issues?